src/sentry/integrations/jira_server/integration.py

```diff
@@ -705,11 +705,17 @@ def get_create_issue_config(self, group, user, **kwargs):
         project_id = params.get("project", defaults.get("project"))
         jira_projects = self.get_projects()
 
-        if not project_id or project_id not in {proj["id"] for proj in jira_projects}:
+        if not project_id:
             project_id = jira_projects[0]["id"]
 
         client = self.get_client()
-        issue_type_choices = client.get_issue_types(project_id)
+        try:
+            issue_type_choices = client.get_issue_types(project_id)
```

Comment:
@ceorourke maybe we should always use the non-cached call for this so we avoid having to do this kind of logic

Reply:
Wouldn't that slow it down though? This should be an edge case and usually the cached project list should be good.

Reply:
@ceorourke maybe we can put this abstraction inside of `get_projects` to make this code a bit cleaner.

Reply:
Hmm it's kinda tricky because I need to update the `project_id` which is used later on as well, I could put it all together and return a tuple of `jira_projects`, `issue_type_choices` and `project_id` but that seems worse than what I have now.

Reply:
I would definitely do it within `get_projects`.

I would consider avoiding the cache but that would require spending time measuring the impact. I think it is negligible bit I don't have evidence for it.